EIPS/eip-820.md

```diff
@@ -331,7 +331,9 @@ The contract has the address above for every chain on which it is deployed.
 <details>
 <summary>Raw metadata of <code>./contracts/ERC820Registry.sol</code></summary>
 <pre>
```

Comment:
Still has an extra `<pre>` tag, no?

Reply:
Nope. L653/L657.

Reply:
Right, the `<pre>` tag is closed, but is it necessary?

The rendered HTML (at least on GitHub) already inserts a `<pre>` tag for fenced code blocks:

![image](https://github.com/ethereum/EIPs/assets/57262657/6d4f6af4-9ed5-4ec5-a81a-a26525d1ac76)

Reply:
I will not be removing the `<pre>` tag, as it is an unrelated modification to a Final EIP